W-1684340 Fixed description of success variable in Search in Google Drive (#425)
mulesoft/docs-rpa: rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-search-in-google-drive.adoc

Array of strings containing the names of the files or folders found
* *Success* 
+

> String indicating if the search was successful and returned no errors (`True`)

Reviewer: ```suggestion
String indicating if the search is successful and returns no errors (`True`)
```

+

> To confirm if the search returned any results, verify if the *Amount found matches* is higher than zero. The *Success* variable returns (`True`) even when there are no results but the search is successful. 

Reviewer: ```suggestion
To confirm if the search returns any results, verify if the *Amount found matches* is higher than zero. The *Success* variable returns (`True`) even if there are no results in a successful search. 
```
